caffe2/CMakeLists.txt

```diff
@@ -1068,6 +1068,10 @@ if(USE_XPU)
       target_link_libraries(torch_xpu PRIVATE
         "-Wl,--whole-archive,\"$<TARGET_FILE:torch_xpu_ops>\" -Wl,--no-whole-archive")
     endif()
+
+    # Set cached ${ATen_XPU_INCLUDE_DIRS} to torch
+    include_directories(SYSTEM ${ATen_XPU_INCLUDE_DIRS})
```

Comment:
How do cmake variables of CPU, CUDA or ATen include directories do in PyTorch? Do we align with them?

Reply:
They use the the the similar way: https://github.com/pytorch/pytorch/blob/ab9791c0e342753013181eeeab300a05774fc456/CMakeLists.txt#L1205-L1206